### PR TITLE
Feature/add rancher

### PR DESCRIPTION
**Pull request by alexhq**
Add a Rancher cookbook, showing the install of Sextant Community edition on a Rancher managed EKS cluster

### CODE REVIEW

File: docs/rancher/index.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+# Rancher Cookbook
+
+Recipe for installing
+_Sextant | Community_ edition on a Rancher managed kubernetes cluster to deploy
```
Comment (duncanjw) [inline docs/rancher/index.md]: Can we lose italics and go with Sextant Community Edition here.

File: docs/rancher/index.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+## Install Prerequisite Tools
+
+You will need the up-to-date versions of the following tools installed -
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+
+## Sign up for an evaluation licence
+
+To install _Sextant | Community_ edition, you will need credentials
```
Comment (duncanjw) [inline docs/rancher/index.md]: Can we lose italics and go with Sextant Community Edition here.

File: docs/rancher/index.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+with the exception of Daml support which is subject to our [Evaluation EULA](https://sextant-resources.s3.amazonaws.com/agreements/Blockchain+Technology+Partners+Limited+(Evaluation)+End+User+License+Agreement.pdf)
+
+Find out more at [blockchaintp.com/sextant](https://www.blockchaintp.com/sextant)
+
+For documentation, visit [docs.blockchaintp.com](https://docs.blockchaintp.com)
+
+## Create a cluster and set up Rancher
+
+We will use an AWS EKS cluster, which will both run Rancher server and act as
+the cluster for our Sextant install.
```
Comment (duncanjw) [inline docs/rancher/index.md]: This is okay

File: docs/rancher/index.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+You should now have Rancher installed on your eksctl cluster and
+accessible via URL
+
+## Install Sextant
+
+Log in to Rancher and select the cluster you want to install Sextant on,
+in our example, this will be the local cluster.
+
+![Local Rancher cluster](../images/rancher/screenshot1.png)
```
Comment (duncanjw) [inline docs/rancher/index.md]: Can you use meaningful screenshot names so if we need to change one we can identity it easily

File: docs/rancher/index.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+
+This will fetch the initial random password for your Sextant install.
+
+!!!Important
+    Make sure you save this password, as it will not be possible to retrieve
+    it if the Sextant deployment is restarted.
+
+![Initial Password](../images/rancher/screenshot7.png)
+
+Now run the second command from the install nodes. In our example this is:
```
Comment (duncanjw) [inline docs/rancher/index.md]: _notes_ not nodes

Comment (alexhq) [reply docs/rancher/index.md]: thanks, corrected

File: docs/rancher/index.md (hunk 4 above)
Comment (alexhq) [reply docs/rancher/index.md]: I've updated them all, but let me know if any don't make sense

File: docs/rancher/index.md (hunk 2 above)
Comment (alexhq) [reply docs/rancher/index.md]: Done

File: docs/rancher/index.md (hunk 1 above)
Comment (alexhq) [reply docs/rancher/index.md]: Done

File: docs/rancher/index.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
+# Rancher Cookbook
+
+Recipe for installing
+Sextant | Community edition on a Rancher managed kubernetes cluster to deploy
```
Comment (duncanjw) [inline docs/rancher/index.md]: Please lose the '|' should be Sextant Community Edition

File: docs/rancher/index.md [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+
+## Install Prerequisite Tools
+
+You will need the up-to-date versions of the following tools installed -
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+
+## Sign up for an evaluation licence
+
+To install Sextant | Community edition, you will need credentials
```
Comment (duncanjw) [inline docs/rancher/index.md]: As above

File: docs/rancher/index.md [hunk 8]
```diff
@@ -0,0 +1,191 @@
+# Rancher Cookbook
+
+Recipe for installing
+Sextant | Community edition on a Rancher managed kubernetes cluster to deploy
+and manage blockchain networks.
+
+## Install Prerequisite Tools
+
+You will need the up-to-date versions of the following tools installed -
```
Comment (duncanjw) [inline docs/rancher/index.md]: Singular ... version of the following tool ...

File: docs/rancher/index.md [hunk 9]
```diff
@@ -0,0 +1,191 @@
...
+
+2 - Get kubeconfig credentials
+
+```bash
+eksctl utils write-kubeconfig --cluster=rancher-server [--kubeconfig=<path>]
+```
+
+You should now have Rancher installed on your eksctl cluster and
+accessible via URL
+
```
Comment (duncanjw) [inline docs/rancher/index.md]: Remove this section. We want these instructions to focus on using Rancher to install Sextant

File: docs/rancher/index.md [hunk 10]
```diff
@@ -0,0 +1,191 @@
+# Rancher Cookbook
+
+Recipe for installing
+Sextant | Community edition on a Rancher managed kubernetes cluster to deploy
+and manage blockchain networks.
+
+## Install Prerequisite Tools
```
Comment (duncanjw) [inline docs/rancher/index.md]: Make this section "Prerequisites" please

Comment (duncanjw) [reply docs/rancher/index.md]: In addition to the note about kubectl please add words to the effect that a prerequisite is SUSE Rancher v2.6 or above

File: docs/rancher/index.md [hunk 11]
```diff
@@ -0,0 +1,191 @@
+# Rancher Cookbook
```
Comment (duncanjw) [inline docs/rancher/index.md]: Officially it is now SUSE Rancher

Comment (duncanjw) [reply docs/rancher/index.md]: Here and in the BTP Cookbooks table of contents please

File: docs/rancher/index.md (hunk 10 above)
Comment (duncanjw) [reply docs/rancher/index.md]: Please include link to SUSE Rancher https://www.suse.com/products/suse-rancher/ for newbies

**Review by duncanjw — CHANGES_REQUESTED** [review]

File: docs/rancher/index.md (hunk 1 above)
Comment (alexhq) [reply docs/rancher/index.md]: should be correct now

File: docs/rancher/index.md (hunk 2 above)
Comment (alexhq) [reply docs/rancher/index.md]: should be correct now

File: docs/rancher/index.md (hunk 6 above)
Comment (alexhq) [reply docs/rancher/index.md]: should be correct now

File: docs/rancher/index.md (hunk 9 above)
Comment (alexhq) [reply docs/rancher/index.md]: done, changed to just the prerequisites section

File: docs/rancher/index.md (hunk 11 above)
Comment (alexhq) [reply docs/rancher/index.md]: Both updated, I've left the directory as rancher, but updated the mkdocs.yaml to show SUSE Rancher

File: docs/rancher/index.md (hunk 10 above)
Comment (alexhq) [reply docs/rancher/index.md]: Done, I've specified versions for both Kubernetes and Rancher

**Review by duncanjw — APPROVED** [review]

**Review by duncanjw — APPROVED** [review]

**Review by duncanjw — APPROVED** [review]

**Review by duncanjw — APPROVED** [review]

**Review by duncanjw — APPROVED** [review]

**Pull request merged**